src/plugins/crypto.h

```diff
@@ -36,9 +37,11 @@ typedef enum {
 
 #define DEFAULT_LUKS_KEYSIZE_BITS 256
 #define DEFAULT_LUKS_CIPHER "aes-xts-plain64"
+#define DEFAULT_LUKS_SECTOR_SIZE 512
```

Comment:
This is for default value when creating luks2 format, so it can be different number than the `SECTOR_SIZE`. Even if sector size of the device is for example 4096, we still have to use 512 when calculating sizes for functions where cryptsetup returns number of sectors. I've renamed the constant to `DEFAULT_LUKS2_SECTOR_SIZE` to make this less confusing.

Reply:
OK, thanks